### PR TITLE
Pass the "certs" directory as part of the URL to Open().

### PR DESCRIPTION
**Pull request by petermattis**

### CODE REVIEW

File: client/db.go [hunk 1]
```diff
@@ -152,8 +152,12 @@ func Open(addr string) *DB {
 		log.Fatal(err)
 	}
 
-	// TODO(pmattis): This isn't right.
-	ctx := testutils.NewTestBaseContext()
+	ctx := &base.Context{}
+	ctx.InitDefaults()
+	q := u.Query()
+	if dir := q["certs"]; len(dir) > 0 {
```
Comment (tschottdorf) [inline client/db.go]: I think you'll have to do `dir != ""` with `go-nyan`.

**Comment by tschottdorf** [comment]
LGTM

File: client/db.go (hunk 1 above)
Comment (spencerkimball) [inline client/db.go]: Where is this parameter documented?

**Comment by spencerkimball** [comment]
LGTM

File: client/db.go (hunk 1 above)
Comment (petermattis) [inline client/db.go]: Documentation? Bah!

Done.

Comment (petermattis) [inline client/db.go]: @tschottdorf `dir` is of type `[]string`.

**Pull request merged**